datahub-web-react/src/app/search/suggestions/SearchQuerySugggester.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+import React from 'react';
+import { useHistory } from 'react-router';
+import { SearchSuggestion } from '../../../types.generated';
+import { navigateToSearchUrl } from '../utils/navigateToSearchUrl';
+import { ANTD_GRAY_V2 } from '../../entity/shared/constants';
+
+const TextWrapper = styled.div`
+    font-size: 14px;
+    color: ${ANTD_GRAY_V2[8]};
+    margin: 16px 0 -8px 32px;
+`;
+
+export const SuggestedText = styled.span`
+    color: ${(props) => props.theme.styles['primary-color']};
+    color: ${(props) => props.theme.styles['primary-color']};
+    text-decoration: underline ${(props) => props.theme.styles['primary-color']};
+    cursor: pointer;
+`;
+
+interface Props {
+    suggestions: SearchSuggestion[];
+}
+
+export default function SearchQuerySuggester({ suggestions }: Props) {
+    const history = useHistory();
+
+    if (suggestions.length === 0) return null;
+    const suggestText = suggestions[0].text;
+
+    function searchForSuggestion() {
+        navigateToSearchUrl({ query: suggestText, history });
+    }
+
+    return (
+        <TextWrapper>
+            Did you mean <SuggestedText onClick={searchForSuggestion}>{suggestText}</SuggestedText>
```

Comment:
Non-blocking. But instead of a clickable span this could be a Button of type "text" I think. I think I remember seeing us use that a few times in the notifications feature. Related, I feel like it may be a bit more semantic if we use Link components and make `navigateToSearchUrl` into a `getSearchUrl`. Less callbacks needed if we do that.

Reply:
yeah I tried that but I just couldn't get it to be styled the way that I wanted it to... but I definitely like turning navigateToSearchUrl as a link that's smooth.